go/libkb/passphrase_stream_cache.go

```diff
@@ -53,10 +53,21 @@ func (s *PassphraseStreamCache) PassphraseStreamRef() *PassphraseStream {
 }
 
 func (s *PassphraseStreamCache) Valid() bool {
+	return s.ValidPassphraseStream() && s.ValidTsec()
```

Comment:
after kex provisioning.  new device has ValidPassphraseStream but not valid 3sec.